CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-29.9.3
+
+This edge release updates the proxy's dependency on the `rustls` library to
+patch security vulnerability [RUSTSEC-2023-0052]  (GHSA-8qv2-5vq6-g2g7), a
```

Comment:
RUSTSEC link is broken

Reply:
The changelog already includes the link and the linter complains if it's declared twice. Might need to declare it with a different name...